### DISCUSSION TITLE
Disable diagnostic hints

### DISCUSSION THREAD

**Discussion by dobryak** [VonHeikemen/lsp-zero.nvim#404] [Q&A] [answered]
Hello. I would like to know if there is a way to completely disable inlay hints? I spent a whole day trying to find a way to disable this ugly feature... Or do I need to patch Vim to remove this? This "feature" is the most useless and ugly thing I have ever seen...

`vim.lsp.inlay_hint.enable(false)` has no effect

Thank you in advance for your help!

**Comment by VonHeikemen** [comment]
Inlay hints are disabled by default in Neovim.

Do you mean inlay hints like "type information" or virtual text that shows error messages?

**Reply by dobryak** [reply]
I mean this:
![image](https://github.com/user-attachments/assets/3ff2f822-ee19-4f49-81d1-b1dd40999efb)

I want to completely disable these hints...

**Reply by VonHeikemen** [reply, marked as answer]
Did you try this?

```lua

vim.diagnostic.config({
  virtual_text = false,
})
```

**Reply by dobryak** [reply]
Oh, yes, this is exactly what I wanted. Thank you a lot! I've been looking for something completely different all this time...
